portal/add_edit_event_user.php

```diff
@@ -487,8 +487,9 @@
     $user = sqlQueryNoLog("SELECT users.username FROM users WHERE authorized = 1 And id = ?", array($_POST['form_provider_ae']));
     $rtn = addPnote($_POST['form_pid'], $note, 1, 1, $title, $user['username'], '', 'New');
 
-    $_SESSION['whereto'] = 'appointmentcard';
-    header('Location:./home.php#appointmentpanel');
+
+    OpenEMR\Common\Session\SessionUtil::setSession('whereto', '#appointmentcard');
```

Comment:
same issue as noted below. if these setSession calls are meant for use in portal, then would change these all to direct session changes or else will be creating a new core OpenEMR session:
```
$_SESSION['whereto'] = '#appointmentcard';
```